pkg/replication/options.go

```diff
@@ -119,6 +125,18 @@ func (o *Options) WithAllowTxDiscarding(allowTxDiscarding bool) *Options {
 	return o
 }
 
+// WithSkipIntegrityCheck disable integrity checks when reading data during replication
+func (o *Options) WithSkipIntegrityCheck(skipIntegrityCheck bool) *Options {
```

Comment:
This should be just WithIntegrityCheck imo, because it can be set to either true or false, adding skip just makes it conforming to skip